### PR TITLE
[tests] Add gerrit and its study sections

### PR DESCRIPTION
**Pull request by valeriocos**
This code includes an example of gerrit and study sections.

### CODE REVIEW

**Comment by sanacl** [comment]
@valeriocos tests failed https://travis-ci.org/chaoss/grimoirelab-sirmordred/builds/574242900?utm_source=github_status&utm_medium=notification

**Comment by valeriocos** [comment]
thank you @sanacl , I'll check it and add you as reviewer when ready

File: tests/test_studies.cfg [hunk 1]
```diff
@@ -90,9 +90,32 @@ no_incremental = true   # default: false
...
+studies = [enrich_demography:gerrit, enrich_onion:gerrit]
+
+[enrich_demography:gerrit]
+date_field = ???         # default: grimoire_creation_date
+author_field = ???       # default: author_uuid
+
+[enrich_onion:gerrit]
+in_index = ???          # default: gerrit_onion-src
+out_index = ???         # default: gerrit_onion-enriched
+data_source = ???       # default : gerrit
```
Comment (sanacl) [inline tests/test_studies.cfg]: There are two extra whitespaces here that should be removed.

```
# default : gerrit
         ^^^
```

File: tests/test_config.py [hunk 2]
```diff
@@ -106,6 +108,20 @@ def test_init_studies(self):
...
+        self.assertIn('author_field', enrich_demography_gerrit_params)
+
+        self.assertIn('enrich_onion:gerrit', top_sections)
+        self.assertIn('in_index', enrich_onion_gerrit_params)
+        self.assertIn('out_index', enrich_onion_gerrit_params)
+        self.assertIn('data_source', enrich_onion_gerrit_params)
+        self.assertIn('contribs_field', enrich_onion_gerrit_params)
+        self.assertIn('timeframe_field', enrich_onion_gerrit_params)
+        self.assertIn('sort_on_field', enrich_onion_gerrit_params)
+        self.assertIn('no_incremental', enrich_onion_gerrit_params)
```
Comment (sanacl) [inline tests/test_config.py]: I miss a check for 'seconds'. Shouldn't it be here?

Comment (valeriocos) [reply tests/test_config.py]: good catch

**Comment by valeriocos** [comment]
please @sanacl check it again

**Pull request merged**